airflow/www_rbac/views.py

```diff
@@ -392,6 +392,33 @@ def task_stats(self, session=None):
                     payload[dag.safe_dag_id].append(d)
         return wwwutils.json_response(payload)
 
+    @expose('/last_dagruns')
+    @has_access
+    @provide_session
+    def last_dagruns(self, session=None):
```

Comment:
RBAC has different templates so nothing is calling this yet.

Reply:
Fixed